gallery/how_to/compile_models/from_coreml.py

```diff
@@ -23,13 +23,12 @@
 
 This article is an introductory tutorial to deploy CoreML models with Relay.
 
-For us to begin with, coremltools module is required to be installed.
-
-A quick solution is to install via pip
+To begin, we must install coremltools:
 
 .. code-block:: bash
 
-    pip install -U coremltools --user
```

Comment:
i'm curious, does `--user` not work with colab? It's not the worst thing to tell folks new to managing python venvs.

Reply:
`--user` works as you would expect in Colab. However, the other tutorials don't use that flag for their dependencies, so I changed this to be consistent. Might be out of scope - what do you think?